### PR TITLE
Google Analytics support

### PR DESCRIPTION
**Pull request by adrtod**
Added Google Analytics support by adding an optional argument `tracking_id` to the `as.sd_package()` function.
It can be used directly as an argument of the `build_site()` function.
This will add the corresponding html code in the footer of each page enabling website audience tracking.
Documented in the `as.sd_package()` function.

### CODE REVIEW

**Comment by adrtod** [comment]
Please ignore this PR. I will redo it as a single commit

**Comment by adrtod** [comment]
Well, I did not find how to do it. But I guess this is on your side.
If not please tell me what I have to do.
Despite github message about conflicts, the diff is ok, there is no conflict.

**Comment by hadley** [comment]
You've made your patch against an old version of the repo

**Comment by adrtod** [comment]
Thanks ! should be ok now

File: R/build.r [hunk 1]
```diff
@@ -38,7 +43,8 @@ build_site <- function(pkg = ".",
                        with_vignettes = TRUE,
                        with_demos = TRUE,
                        with_readme = TRUE,
-                       launch = interactive()
+                       launch = interactive(),
+                       tracking_id = NULL
```
Comment (hadley) [inline R/build.r]: Why not just call this `ganalytics_id` so you only need one variable? (Not both `tracking_id` and `ganalytics`?  That would eliminate most of the code below.

Comment (adrtod) [inline R/build.r]: Sure

**Comment by adrtod** [comment]
Done

**Comment by hadley** [comment]
Closing for now. Feel free to reopen in a week or so, once the major rewrite has been accomplished. It will be much more natural to include this as a key in `_staticdocs.yaml`. See https://github.com/hadley/staticdocs/blob/master/vignettes/staticdocs.Rmd for the whole plan.

**Comment by adrtod** [comment]
Reopened in #169

**Pull request closed without merging**